README.md

```diff
@@ -1,35 +1,88 @@
-# catppucicn webring
+<h3 align="center">
+	<img src="https://raw.githubusercontent.com/catppuccin/catppuccin/main/assets/logos/exports/1544x1544_circle.png" width="100" alt="Logo"/><br/>
+	<img src="https://raw.githubusercontent.com/catppuccin/catppuccin/main/assets/misc/transparent.png" height="30" width="0px"/>
+	The Catppuccin <a href="https://en.wikipedia.org/wiki/Webring">Webring</a>
+	<img src="https://raw.githubusercontent.com/catppuccin/catppuccin/main/assets/misc/transparent.png" height="30" width="0px"/>
+</h3>
 
-## How to Join
-The only required information is your site's **URL** and your desired **slug**.
+<p align="center">
+	<a href="https://github.com/isabelroses/catppuccin-webring/stargazers"><img src="https://img.shields.io/github/stars/isabelroses/catppuccin-webring?colorA=363a4f&colorB=b7bdf8&style=for-the-badge"></a>
+	<a href="https://github.com/isabelroses/catppuccin-webring/issues"><img src="https://img.shields.io/github/issues/isabelroses/catppuccin-webring?colorA=363a4f&colorB=f5a97f&style=for-the-badge"></a>
+	<a href="https://github.com/isabelroses/catppuccin-webring/contributors"><img src="https://img.shields.io/github/contributors/isabelroses/catppuccin-webring?colorA=363a4f&colorB=a6da95&style=for-the-badge"></a>
+</p>
 
-Optionally, you can also include the following for your site listing:
+<p align="center">
+	<img src="assets/previews/preview.webp"/>
+</p>
 
-- Full **title** of your site
-- A brief **description** of your site
-- Some form of **contact info** (email, fedi, etc)
-- If you have one, your **RSS** link
+## Previews
 
-Include your information by adding a new entry to *`websites.json`*
+<details>
+<summary>🌻 Latte</summary>
+<img src="assets/previews/latte.webp"/>
+</details>
+<details>
+<summary>🪴 Frappé</summary>
+<img src="assets/previews/frappe.webp"/>
+</details>
+<details>
+<summary>🌺 Macchiato</summary>
+<img src="assets/previews/macchiato.webp"/>
+</details>
+<details>
+<summary>🌿 Mocha</summary>
+<img src="assets/previews/mocha.webp"/>
+</details>
 
-### JSON Example
+## Usage
 
-```json
-{
-  "name": "Example Website",
-  "slug": "example",
-  "about": "I'm an example!!!",
-  "url": "https://example.com/",
-  "rss": "https://example.com/index.xml",
-  "owner": "person@example.com"
-}
-```
+1. Go to...

```

Comment:
Remove this please

Reply:
Remove what? The placeholder? The website wasn't working yet soo....

Reply:
Oh all the previews? We should have previews at some point no?

Reply:
I mean, it feels weird to me.